client/src/app/pages/controls/business-services/business-services.tsx

```diff
@@ -83,7 +83,8 @@ export const BusinessServices: React.FC = () => {
 
   const tableControls = useLocalTableControls({
     tableName: "business-services-table",
-    idProperty: "name",
+    idProperty: "id",
```

Comment:
good catch! please mention those fixes in the commit message (or extract to a small PR)?

Reply:
> good catch! please mention those fixes in the commit message (or extract to a small PR)?

I mentioned that kind of change in the description as "and if necessary, adjust the idProperty".  Could be called out more explicitly.

Also interesting that it really didn't do anything bad or wrong with any behavior.  Apparently the name on the entity is enforced unique so it worked just fine.  But now both bits of info are available on the data attributes. :-)